### ISSUE TITLE
Problem on IE 10, 11, Edge

### ISSUE THREAD

**Issue by Draccan** [cezary/react-loading#19]
Hi,

I have a problem with your component.
I use your loading component in this way: <Loading type={"spin"} color='#666666'/>.

While the Loading spins/rotates on Firefox and Chrome, it doesn't rotate on Internet Explorer 10,11 and Edge.

Could you resolve this problem please?

**Comment by dilipvinu** [comment]
SVG animations don't seem to be supported on IE 11 and below, might have to use a fallback.

**Comment by kacimimoghite** [comment]
@Draccan did you resolve the problem ?

**Comment by Draccan** [comment]
@kacimimoghite Yes, I've used some polyfills if I remember correctly. However I don't work in the company where I was facing this problem so I can't be sure about it.

**Comment by adityaghag** [comment]
Have anyone resolved the issue?

**Comment by fakiolinho** [comment]
No, unfortunately not @adityaghag

**Comment by DenRostokin** [comment]
I`ve resoleved the problem with this polyfill:
https://github.com/ericwilligers/svg-animation
